serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: imputation
+
+provider:
+  name: aws
+  runtime: python3.7
+  region: eu-west-2
+
+functions:
```

Comment:
Will need to be updated to include all functions when serverless is properly implemented.